stdlib/http/src/main/ballerina/src/http/cookie/csv_persistent_cookie_handler.bal

```diff
@@ -0,0 +1,254 @@
+    #
+    # + name - Name of the persistent cookie to be removed
+    # + domain - Domain of the persistent cookie to be removed
+    # + path - Path of the persistent cookie to be removed
+    # + return - An error will be returned if there is any error occurred during removing the cookie or else nil is returned
+    public function removeCookie(string name, string domain, string path) returns @tainted CookieHandlingError? {
+        cookieNameToRemove = name;
+        cookieDomainToRemove = domain;
+        cookiePathToRemove = path;
+        CookieHandlingError err;
+        if (file:exists(self.fileName)) {
+            if(!self.cookiesTable.hasNext()) {
+                var tblResult = readFile(self.fileName);
+                if (tblResult is table<myCookie>) {
+                    self.cookiesTable = tblResult;
+                } else {
+                    err = error(COOKIE_HANDLING_ERROR, message = "Error in reading the csv file", cause = tblResult);
+                    return err;
+                }
+            }
+            int | error count = self.cookiesTable.remove(checkRemoveCriteria);
+            if (count is error || count <= 0) {
+                err = error(COOKIE_HANDLING_ERROR, message = "Error in removing cookie: No such cookie to remove");
+                return err;
+            }
+            error? removeResults = file:remove(<@untainted> self.fileName);
+            if (removeResults is error) {
+                err = error(COOKIE_HANDLING_ERROR, message = "Error in removing the csv file", cause = removeResults);
+                return err;
+            }
+            var writeResult = writeToFile(self.cookiesTable, <@untainted> self.fileName);
+            if (writeResult is error) {
+                err = error(COOKIE_HANDLING_ERROR, message = "Error in writing the csv file", cause = writeResult);
+                return err;
+            }
+            return;
+        }
+        err = error(COOKIE_HANDLING_ERROR, message = "Error in removing cookie: No persistent cookie store file to remove");
+        return err;
+    }
+
+    # Removes all persistent cookies.
+    #
+    # + return - An error will be returned if there is any error occurred during removing all the cookies or else nil is returned
+    public function removeAllCookies() returns CookieHandlingError? {
+        error? removeResults = file:remove(self.fileName);
+        if (removeResults is error) {
+            CookieHandlingError err = error(COOKIE_HANDLING_ERROR, message = "Error in removing the csv file", cause = removeResults);
+            return err;
+        }
+    }
+};
+
+function validateFileExtension(string fileName) returns string|CookieHandlingError {
+    if (fileName.toLowerAscii().endsWith(".csv")) {
+        return fileName;
+    }
+    CookieHandlingError err = error(COOKIE_HANDLING_ERROR, message = "Invalid file format");
+    return err;
+}
+
+function readFile(string fileName) returns @tainted error | table<myCookie> {
```

Comment:
```suggestion
function readFile(string fileName) returns @tainted error|table<myCookie> {
```

Reply:
Can we fix other similar occurences as well? It's better to remove spaces around `|`